.gitignore

```diff
@@ -51,3 +51,6 @@ flycheck-*
 
 ### vim
 .ropeproject/
+
+### env path
+/env/
```

Comment:
I don't think that mongomock creates such a folder. Please remove that and add it to your personal .gitignore

Reply:
Sure.